src/assets.ts

```diff
@@ -1,7 +1,9 @@
 import { ensureDir, extname, mime, walk } from "./deps.ts";
+import RouteHandler from "./routeHandler.ts";
 
 const assets = async (dir: string) => {
   const meta = {
+    apiRoutes: RouteHandler,
```

Comment:
The route handlers are collected in a singleton tree. This is not quite as performant as the Map for the other collections, but I tried to make any optimizations possible.